doc/development_setup.md

```diff
@@ -0,0 +1,34 @@
+# Development Setup
+
+This document should get you up to speed and give you a working development environment for
+`transcribee`. The instructions in this document may not be used for production installations!
+
+The transcribee project uses a development-setup managed by `nix`. This allows us to all use the
+same software & versions conviniently. Thus, [you need to have a working nix installation on your
+system](https://nix.dev/tutorials/install-nix).
+
+## Minimal setup
+
+If you just want to try out `transcribee`, you need to go through the following steps:
+
+1. [Install `nix` on your system](https://nix.dev/tutorials/install-nix).
```

Comment:
This is ~10x worse than "just install docker"? :D I think we should mark this as subject to change.